src/axom/core/utilities/Timer.hpp

```diff
@@ -18,27 +18,12 @@
 #define TIMER_HPP_
 
 #include "axom/config.hpp"
-#ifdef AXOM_USE_CXX11
-  #include "axom/core/detail/ChronoTimer.hpp"
-#else
-  #ifdef WIN32
-    #include "axom/core/detail/TickCountTimer.hpp"
-  #else
-    #include "axom/core/detail/TimeofdayTimer.hpp"
```

Comment:
I think it is safe to remove ``TickCountTimer.hpp`` and ``TimeofdayTimer.hpp``, since we are not using them. We could also perhaps consolidate the ``ChronoTimer.hpp`` in ``Timer.hpp`` to simplify the code in the future.  Most of this was done to support non C++11 compilers.

Reply:
Done.